Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/DisplayMtl.mm

```diff
@@ -1401,6 +1408,12 @@ bool IsMetalDisplayAvailable()
     return false;
 #endif
 }
+
+bool DisplayMtl::supportsVariableRasterizationRate() const
+{
+    return [mMetalDevice supportsRasterizationRateMapWithLayerCount:1];
```

Comment:
Upstream ANGLE would need a macOS version check around this call, since Chrome still ships on macOS 10.13 but this API was only introduced in macOS 10.15.4+.